packages/cli/src/cmds/search/search.ts

```diff
@@ -169,8 +168,7 @@ export const handler = async (argv: any) => {
     const options: SearchOptions = {
       maxResults,
     };
-    const appmapDir = await locateAppMapDir();
-    const response = await AppMapIndex.search(appmapDir, query, options);
+    const response = await AppMapIndex.search([process.cwd()], query, options);
```

Comment:
It seems wrong that `configuration.directories` isn't respected here? Is this RPC function used by the frontend?

Reply:
The `search` RPC method isn't called directly from the frontend as far as I'm aware, only via `explain`. In that context, it does look like it's using the correct directories.

It also looks correct when calling the `search` RPC method
https://github.com/getappmap/appmap-js/blob/eb4b88c049a040e395455ee0bb0d24730b9ec452/packages/cli/src/rpc/search/search.ts#L60